### ISSUE TITLE
chrome/toggle_tabs_toolbar_with_alt.css not working on macOS

### ISSUE THREAD

**Issue by matru** [MrOtherGuy/firefox-csshacks#396]
Not sure if the issues come from that the menubar seems to be always displayed (not natively through firefox, but natively to the macOS top bar), meaning basically there's no menubar toggle as there is in windows. Is there any workaround this?

**Comment by MrOtherGuy** [comment]
Unlikely. Anything that deals with menubar is highly likely to simply not work in macOS because as yo say Firefox doesn't have it's own menubar there.

That style only works because Alt key makes menubar temporarily show up, and we simply make tabs toolbar visibility linked to the visibility of menubar. But if menubar doesn't exist then you would need to have something else that can be toggled which visibility tabs toolbar could reflect.

**Comment by matru** [comment]
Precisely. Sadly, with the standard setup I don't have anything in mind that can mimic alt behavavior that windows users have.
